features/karaf/openhab-addons/src/main/feature/feature.xml

```diff
@@ -859,6 +859,15 @@
         <bundle start-level="80">mvn:org.openhab.addons.bundles/org.openhab.binding.windcentrale/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-binding-xmppclient" description="XMPP Client Binding" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
+        <bundle dependency="true" start-level="80">mvn:org.igniterealtime.smack/smack-java7/4.3.1</bundle>
```

Comment:
The compilation itself worked on travis but the feature verification failed. Please remove the start-level attributes from the dependencies. And add "wrap" to the URI like in "wrap:mvn:org.igniterealtime.smack/smack-java7/4.3.1" (and for the other deps as well)

Reply:
Done. But I don’t understand how the DCO check works...